support-frontend/assets/helpers/productPrice/subscriptions.ts

```diff
@@ -10,13 +10,14 @@ import type {
 	OphanComponentEvent,
 	OphanComponentType,
 } from '../tracking/ophan';
+
 // ----- Types ------ //
-const DigitalPack: 'DigitalPack' = 'DigitalPack';
-const PremiumTier: 'PremiumTier' = 'PremiumTier';
-const DailyEdition: 'DailyEdition' = 'DailyEdition';
-const GuardianWeekly: 'GuardianWeekly' = 'GuardianWeekly';
-const Paper: 'Paper' = 'Paper';
-const PaperAndDigital: 'PaperAndDigital' = 'PaperAndDigital';
+const DigitalPack = 'DigitalPack' as const;
+const PremiumTier = 'PremiumTier' as const;
+const DailyEdition = 'DailyEdition' as const;
+const GuardianWeekly = 'GuardianWeekly' as const;
+const Paper = 'Paper' as const;
+const PaperAndDigital = 'PaperAndDigital' as const;
```

Comment:
ESLint was adamant this is the way to go. The [rule details](https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/prefer-as-const.md) for 'prefer usage of `as const` over literal type' seems to apply here.

![image](https://user-images.githubusercontent.com/11380557/151544332-34f40b9a-15b1-4104-9b24-32e21992abe2.png)

Reply:
Yeah I think this is better!